### PR TITLE
kubeadm: update kubeadm banner to beta

### PR DESCRIPTION
**Pull request by dmmcquay**
**What this PR does / why we need it**: Updates the intro banner for kubeadm, which used to  state it is in alpha (but we are going to beta). This also updates the tagged github group (one that no longer exists) to the sig-cluster-lifecycle-misc group.  

**Special notes for your reviewer**: /cc @jbeda 

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: cmd/kubeadm/app/cmd/cmd.go [hunk 1]
```diff
@@ -35,11 +35,11 @@ func NewKubeadmCommand(f cmdutil.Factory, in io.Reader, out, err io.Writer) *cob
 			kubeadm: easily bootstrap a secure Kubernetes cluster.
 
 			    ┌──────────────────────────────────────────────────────────┐
-			    │ KUBEADM IS ALPHA, DO NOT USE IT FOR PRODUCTION CLUSTERS! │
+			    │ KUBEADM IS BETA, DO NOT USE IT FOR PRODUCTION CLUSTERS! │
```
Comment (jbeda) [inline cmd/kubeadm/app/cmd/cmd.go]: It looks like this doesn't line up...

**Review by jbeda — CHANGES_REQUESTED** [review]

**Comment by jbeda** [comment]
LGTM @dmmcquay but I did a quick grep through the kubeadm dir and found 2 more references to alpha.  Mind getting those in this too?

* https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/init.go#L136
* https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/cmd/join.go#L136

Sorry for trickling this in but I figure at this point it pays to be complete.

**Comment by dmmcquay** [comment]
Good catch... I tried to do the same thing but I think I search case sensitive and missed those...

**Review by jbeda — APPROVED** [review]

**Pull request merged**